src/components/Shared/Details.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, ReactNode } from "react";
+import styled from "@emotion/styled";
+
+import colors from "../../styles/colors";
+
+const Summary = styled.summary<{ hasBorderBottom?: boolean }>((props) => ({
+  "&::before": {
+    // remove mozilla protocol +/- icon
+    content: "none",
+  },
+  paddingRight: 0,
+  paddingBottom: props.hasBorderBottom ? 8 : 0,
+  borderBottom: props.hasBorderBottom ? `1px solid ${colors.grey}` : 0,
+}));
+
+interface DetailsProps {
+  summaryContent: ReactNode;
+  hiddenContent: ReactNode;
+  defaultOpen: boolean;
+  hasBorderBottom?: boolean;
+}
+
+const Details: FC<DetailsProps> = ({
+  summaryContent,
+  hiddenContent,
+  defaultOpen,
+  hasBorderBottom,
+}: DetailsProps) => {
+  return (
+    <details open={defaultOpen}>
+      <Summary hasBorderBottom={hasBorderBottom}>{summaryContent}</Summary>
+      {hiddenContent}
+    </details>
+  );
+};
+
```

Comment:
Would love a quick comment here to make sure I am following. This is basically the default HTML `details` element just prefilled / fixed to match the rest of our + mozilla styling practices?

Reply:
Yes, creating a react component to make it reusable. Don't want to keep removing the +/- sign on the right side for every use of `details` (https://protocol.mozilla.org/patterns/molecules/details.html)